src/main/java/com/ozwillo/socatelgraphql/repository/PostRepository.java

```diff
@@ -1,186 +1 @@
        }

        return Optional.empty();
    }

    private GraphPattern buildPostGraphPattern(Variable post, Optional<String> identifier) {
        TriplePattern triplePattern = post.isA((SOCATEL.iri("Post")));
        identifier.ifPresent(s -> triplePattern.andHas(SOCATEL.iri("identifier"), s));
        triplePattern.andHas(SOCATEL.iri("identifier"), var("identifier"))
                .andHas(SOCATEL.iri("description"), var("description"))
                .andHas(SOCATEL.iri("creationDate"), var("creationDate"))
                .andHas(SOCATEL.iri("language"), var("language"))
                .andHas(SOCATEL.iri("num_likes"), var("num_likes"))
                .andHas(SIOC.iri("num_replies"), var("num_replies"));

        return triplePattern;
    }

    private GraphPattern buildLocationGraphPattern(Variable post) {
        Variable location = var("location");
        return post.has(SOCATEL.iri("location"), var("location"))
                .and(location.has(GN.iri("name"), var("location_name"))
                        .andHas(GN.iri("alternateName"), var("location_alternateName"))
                        .andHas(GN.iri("countryCode"), var("location_countryCode"))).optional();
    }

    private GraphPattern buildOwnerGraphPattern(Variable post) {
        Variable owner = var("owner");
        return post.has(SIOC.iri("has_owner"), var("owner"))
                .and(owner.has(SOCATEL.iri("identifier"), var("owner_identifier"))
                        .andHas(SOCATEL.iri("title"), var("owner_title"))
                        .andHas(SOCATEL.iri("description"), var("owner_description"))
                        .andHas(SOCATEL.iri("webLink"), var("owner_webLink"))
                        .andHas(SOCATEL.iri("language"), var("owner_language"))
                        .andHas(SOCATEL.iri("num_likes"), var("owner_num_likes"))).optional();
    }

    private GraphPattern buildCreatorGraphPattern(Variable post) {
        Variable creator = var("creator");
        return post.has(SOCATEL.iri("createdBy"), var("creator"))
                .and(creator.has(FOAF.iri("name"), var("creator_name"))
                        .andHas(FOAF.iri("username"), var("creator_username"))).optional();
    }

    private SelectQuery buildPostSelectQuery() {
        return Queries.SELECT()
                .prefix(SOCATEL, RDF, SIOC, GN, FOAF)
                .select(var("post"), var("identifier"), var("description"),
                        var("creationDate"), var("language"), var("num_likes"),
                        var("num_replies"), var("location_name"), var("location_alternateName"),
                        var("location_countryCode"), var("owner_identifier"), var("owner_title"),
                        var("owner_description"), var("owner_webLink"), var("owner_language"),
                        var("owner_num_likes"), var("creator_name"), var("creator_username"));
    }
}

```

Comment:
😨

Reply:
what is this ^^

Reply:
I think is my git client, because on my editor I don't have same thing